app/models/course/achievement.rb

```diff
@@ -5,4 +5,8 @@ class Course::Achievement < ActiveRecord::Base
   belongs_to :course, inverse_of: :achievements
 
   default_scope { order(weight: :asc) }
+
+  has_attached_file :icon, :styles => { :medium => "300x300>", :thumb => "100x100>" },
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.